spec/lib/submission_builder/ty2024/states/nj/nj_return_xml_spec.rb

```diff
@@ -8,11 +8,18 @@
     let!(:submission_efile_device_info) { create :state_file_efile_device_info, :submission, :filled, intake: intake }
     let(:build_response) { described_class.build(submission, validate: true) }
     let(:xml) { Nokogiri::XML::Document.parse(build_response.document.to_xml) }
-
+    
     after do
       expect(build_response.errors).not_to be_present
     end
 
+    it "does not include the xmlns attribute on any element in the demo env" do
+      allow(Rails.env).to receive(:demo?).and_return true
+      nodes_with_namespace = xml.document.xpath('//*[namespace-uri()="http://www.irs.gov/efile"]')
+      expect(nodes_with_namespace.count).to eq(0)
+      # Do not check for build errors because schema cannot be validated against without namesapce
```

Comment:
[dust] From my reading, `expect(build_response.errors).not_to be_present` would run after this test, so this comment would not be adhered to... am I missing something?

Reply:
Good point, moved!